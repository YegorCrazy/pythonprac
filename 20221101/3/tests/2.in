a = Grange(2, 2, 20)
print(a)
print(len(a))
if not a:
    print("Empty")
print(*[a[i] for i in range(6)], sep=' ')
print(*list(a), sep=' ')
print(*list(a[2:10]), sep=' ')
print(*list(a[2:20:2]), sep=' ')
