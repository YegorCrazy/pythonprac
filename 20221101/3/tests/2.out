grange(2, 2, 20)
4
2 4 8 16 32 64
2 4 8 16
2 4 8
2 8
